2018-09-13 00:29:31,060 [vivoX9] DEBUG [gby.testng.testcase.InitDriverCase.f() Line: 38] - deviceName: vivoX9
2018-09-13 00:29:31,297 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.getBufferedReader() Line: 84] - 执行命令：[cmd, /c, adb connect 192.168.0.100:5555]
2018-09-13 00:29:31,457 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.runCommandThruProcess() Line: 41] - 命令结果：already connected to 192.168.0.100:5555
2018-09-13 00:29:34,031 [vivoX9] INFO  [gby.appium.listern.TestngRetry.retry() Line: 21] - 测试类：'gby.appium.listern.TestngRetry' 中的测试方法：'f' 执行失败，重试第 1 次
2018-09-13 00:29:34,036 [vivoX9] ERROR [gby.appium.listern.TestNGListener.onTestSkipped() Line: 59] - 【f】测试用例被跳过
2018-09-13 00:29:34,164 [vivoX9] INFO  [gby.appium.listern.TestNGListener.onTestStart() Line: 34] - 【f】测试用例开始执行
2018-09-13 00:29:34,166 [vivoX9] DEBUG [gby.testng.testcase.InitDriverCase.f() Line: 38] - deviceName: vivoX9
2018-09-13 00:29:34,177 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.getBufferedReader() Line: 84] - 执行命令：[cmd, /c, adb connect 192.168.0.100:5555]
2018-09-13 00:29:34,313 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.runCommandThruProcess() Line: 41] - 命令结果：already connected to 192.168.0.100:5555
2018-09-13 00:29:36,705 [vivoX9] ERROR [gby.appium.listern.TestNGListener.onTestFailure() Line: 49] - 【f】测试用例执行失败
2018-09-13 00:29:36,715 [vivoX9] ERROR [gby.appium.listern.TestNGListener.onTestFailure() Line: 49] - 【f】测试用例执行失败
2018-09-13 00:29:36,717 [vivoX9] INFO  [gby.appium.listern.TestNGListener.onFinish() Line: 66] - 测试执行完毕
2018-09-13 00:36:16,957 [vivoX9] DEBUG [gby.testng.testcase.InitDriverCase.f() Line: 38] - deviceName: vivoX9
2018-09-13 00:36:17,127 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.getBufferedReader() Line: 84] - 执行命令：[cmd, /c, adb connect 192.168.0.100:5555]
2018-09-13 00:36:17,303 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.runCommandThruProcess() Line: 41] - 命令结果：already connected to 192.168.0.100:5555
2018-09-13 00:36:18,947 [vivoX9] INFO  [gby.appium.listern.TestngRetry.retry() Line: 21] - 测试类：'gby.appium.listern.TestngRetry' 中的测试方法：'f' 执行失败，重试第 1 次
2018-09-13 00:36:18,953 [vivoX9] ERROR [gby.appium.listern.TestNGListener.onTestSkipped() Line: 59] - 【f】测试用例被跳过
2018-09-13 00:36:19,026 [vivoX9] INFO  [gby.appium.listern.TestNGListener.onTestStart() Line: 34] - 【f】测试用例开始执行
2018-09-13 00:36:19,027 [vivoX9] DEBUG [gby.testng.testcase.InitDriverCase.f() Line: 38] - deviceName: vivoX9
2018-09-13 00:36:19,044 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.getBufferedReader() Line: 84] - 执行命令：[cmd, /c, adb connect 192.168.0.100:5555]
2018-09-13 00:36:19,258 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.runCommandThruProcess() Line: 41] - 命令结果：already connected to 192.168.0.100:5555
2018-09-13 00:36:20,817 [vivoX9] ERROR [gby.appium.listern.TestNGListener.onTestFailure() Line: 49] - 【f】测试用例执行失败
2018-09-13 00:36:20,834 [vivoX9] ERROR [gby.appium.listern.TestNGListener.onTestFailure() Line: 49] - 【f】测试用例执行失败
2018-09-13 00:36:20,837 [vivoX9] INFO  [gby.appium.listern.TestNGListener.onFinish() Line: 66] - 测试执行完毕
2018-09-13 00:46:21,437 [vivoX9] DEBUG [gby.testng.testcase.InitDriverCase.f() Line: 38] - deviceName: vivoX9
2018-09-13 00:46:21,615 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.getBufferedReader() Line: 84] - 执行命令：[cmd, /c, adb connect 192.168.0.100:5555]
2018-09-13 00:46:21,703 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.runCommandThruProcess() Line: 41] - 命令结果：already connected to 192.168.0.100:5555
2018-09-13 00:46:24,224 [vivoX9] INFO  [gby.appium.listern.TestngRetry.retry() Line: 21] - 测试类：'gby.appium.listern.TestngRetry' 中的测试方法：'f' 执行失败，重试第 1 次
2018-09-13 00:46:24,228 [vivoX9] ERROR [gby.appium.listern.TestNGListener.onTestSkipped() Line: 59] - 【f】测试用例被跳过
2018-09-13 00:46:24,319 [vivoX9] INFO  [gby.appium.listern.TestNGListener.onTestStart() Line: 34] - 【f】测试用例开始执行
2018-09-13 00:46:24,324 [vivoX9] DEBUG [gby.testng.testcase.InitDriverCase.f() Line: 38] - deviceName: vivoX9
2018-09-13 00:46:24,349 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.getBufferedReader() Line: 84] - 执行命令：[cmd, /c, adb connect 192.168.0.100:5555]
2018-09-13 00:46:24,585 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.runCommandThruProcess() Line: 41] - 命令结果：already connected to 192.168.0.100:5555
2018-09-13 00:46:27,005 [vivoX9] ERROR [gby.appium.listern.TestNGListener.onTestFailure() Line: 49] - 【f】测试用例执行失败
2018-09-13 00:46:27,012 [vivoX9] ERROR [gby.appium.listern.TestNGListener.onTestFailure() Line: 49] - 【f】测试用例执行失败
2018-09-13 00:46:27,014 [vivoX9] INFO  [gby.appium.listern.TestNGListener.onFinish() Line: 66] - 测试执行完毕
2018-09-13 00:58:19,137 [vivoX9] DEBUG [gby.testng.testcase.InitDriverCase.f() Line: 38] - deviceName: vivoX9
2018-09-13 00:58:19,307 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.getBufferedReader() Line: 84] - 执行命令：[cmd, /c, adb connect 192.168.0.100:5555]
2018-09-13 00:58:23,020 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.runCommandThruProcess() Line: 41] - 命令结果：already connected to 192.168.0.100:5555
2018-09-13 01:00:25,697 [vivoX9] INFO  [gby.appium.listern.TestngRetry.retry() Line: 21] - 测试类：'gby.appium.listern.TestngRetry' 中的测试方法：'f' 执行失败，重试第 1 次
2018-09-13 01:00:25,711 [vivoX9] ERROR [gby.appium.listern.TestNGListener.onTestSkipped() Line: 59] - 【f】测试用例被跳过
2018-09-13 01:00:25,778 [vivoX9] INFO  [gby.appium.listern.TestNGListener.onTestStart() Line: 34] - 【f】测试用例开始执行
2018-09-13 01:00:25,778 [vivoX9] DEBUG [gby.testng.testcase.InitDriverCase.f() Line: 38] - deviceName: vivoX9
2018-09-13 01:00:25,787 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.getBufferedReader() Line: 84] - 执行命令：[cmd, /c, adb connect 192.168.0.100:5555]
2018-09-13 01:00:26,523 [vivoX9] DEBUG [gby.appium.utils.CommandPromptUtil.runCommandThruProcess() Line: 41] - 命令结果：already connected to 192.168.0.100:5555
2018-09-13 01:00:28,926 [vivoX9] ERROR [gby.appium.listern.TestNGListener.onTestFailure() Line: 49] - 【f】测试用例执行失败
2018-09-13 01:00:28,944 [vivoX9] ERROR [gby.appium.listern.TestNGListener.onTestFailure() Line: 49] - 【f】测试用例执行失败
2018-09-13 01:00:28,948 [vivoX9] INFO  [gby.appium.listern.TestNGListener.onFinish() Line: 66] - 测试执行完毕
